benchmarks/dynamo/torchbench.py

```diff
@@ -282,6 +282,10 @@ def load_model(
         if self.args.accuracy and model_name in MAX_BATCH_SIZE_FOR_ACCURACY_CHECK:
             batch_size = min(batch_size, MAX_BATCH_SIZE_FOR_ACCURACY_CHECK[model_name])
 
+        # See https://github.com/pytorch/benchmark/issues/1560
+        if model_name == "speech_transformer":
+            batch_size = 10
```

Comment:
por que 10

Reply:
That's empirically what the batch size torchbench pops out is